code/script.js

```diff
@@ -1,51 +1,252 @@
+const chatContainer = document.getElementById('js-chat-container')
 
 // If you need any global variables that you can use across different functions, declare them here:
 
+//let userEmail = '';
+let userName = '';
+const choice = '';
+const yourPrice = '';
 
 // Declare your functions after this comment
 
-// This function will add a chat bubble in the correct place based on who the sender is
+// This function will add a chat bubble in the correct place based on who the sender is. I think?
 const showMessage = (message, sender) => {
-  // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
+
   if (sender === 'user') {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/user1.png" alt="User" />  
       </section>
     `
-    // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
-  } else if (sender === 'bot') {
+  }
+  else if (sender === 'bot') {
+    console.log(sender);
+    console.log(message);
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/bot1.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
-      </section>
-    `
+      </section>`
   }
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
 // Starts here
+// here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
+
 const greetUser = () => {
-  // here we call the function showMessage, that we declared earlier with the argument "Hello there, What's your name?" for message, and the argument "bot" for sender
-  showMessage("Hello there, What's your name?", 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Hello there!`, 'bot')
+  showMessage(`I'm Anna, a chatbot.`, 'bot')
+  showMessage(`What's your name?`, 'bot')
+}
+
+const userNameInput = event => {
+  event.preventDefault();
+  //I think the name is saved for later, but is it saved globally?
+  userName = nameInput.value;
+  nameInput.value = "";
+  showMessage(` ${userName}`, "user");
+  setTimeout(() => reply(userName), 500);
+  setTimeout(() => askForChoise(), 1200);
+};
+
+
+//Bot's welcome message with username pops up. Asks user to make a choise.
+function reply(firstReply) {
+  showMessage(`Welcome to 'Elbines-Plants' ${userName}!`, 'bot')
+}
+
+//I wanted a delay between the messages, thats why they are in two functions
+const askForChoise = (YesNo) => {
+  //Do I need to type anything in ()?
```

Comment:
You are talking about function askForChoise =(yesNo)=>{} ?? 
If you need more parameters, then you can add as much as you need. But if you don't need it, then you don't need any.